.env.example

```
@@ -1,7 +1,17 @@
+DB_PASSWORD=change-me
+AIRFLOW_DB_USER=airflow
+AIRFLOW_DB_NAME=airflow
+AIRFLOW_DB_PASSWORD=change-me
 SPRING_PROFILE=local
+JWT_SECRET=change-me-at-least-32-characters-long
+CORS_ALLOWED_ORIGINS=http://localhost:3000,http://localhost:5173,http://localhost:5174,http://localhost:5175,http://localhost:5176
+AIRFLOW_UID=50000
+AIRFLOW__CORE__FERNET_KEY=<generate-a-valid-fernet-key>
+AIRFLOW__API__SECRET_KEY=change-me
+AIRFLOW__API_AUTH__JWT_SECRET=change-me
+AIRFLOW_SIMPLE_ADMIN_PASSWORD=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AIRFLOW_SIMPLE_ADMIN_PASSWORD key should go before the AIRFLOW_UID key

(UnorderedKey)